hyprpm/src/core/PluginManager.cpp

```diff
@@ -50,17 +50,26 @@ static std::string getTempRoot() {
     return STR;
 }
 
-SHyprlandVersion CPluginManager::getHyprlandVersion() {
-    static SHyprlandVersion ver;
-    static bool             once = false;
+SHyprlandVersion CPluginManager::getHyprlandVersion(bool running = true) {
```

Comment:
default params in the header pls not source

Furthermore, why is it true by default when it looks to me like you use false more?

Reply:
funnily enough, i had it in the header before. no idea why i moved it

its true by default as that corresponds to the default behavior that the function had before to not accidentally cause incorrect behaviour somewhere else if i missed some statement

Reply:
ok